src/client/metadataApiRetrieve.ts

```diff
@@ -236,6 +244,11 @@ export class MetadataApiRetrieve extends MetadataTransfer<MetadataApiRetrieveSta
       requestBody.singlePackage = this.options.singlePackage;
     }
 
+    // Debug output for API version used for retrieve
+    const manifestVersion = manifestData.version ?? requestBody.apiVersion;
```

Comment:
see above...there's definitely something in manifestData.version

Reply:
Yes, you're correct.  This is a "suspenders and a belt" bit of code that I can modify to just use `manifestData.version`.  I'll make that change.